src/inmanta/loader.py

```diff
@@ -257,6 +260,33 @@ def touch_inits(directory: str) -> None:
                 self._load_module(module_source.name, module_source.hash_value)
 
 
+class PluginModuleLoadException(Exception):
+    """
+    Wrapper exception raised when an exception occurs during plugin module loading.
+    """
+
+    def __init__(self, cause: Exception, module: str, path: str, lineno: Optional[int]) -> None:
+        self.cause: Exception = cause
```

Comment:
is there a particular reason not to use `__cause__` 

https://www.python.org/dev/peps/pep-3134/

Reply:
I felt it made more sense to store the cause explicitly so that the (plain) fields of this class represent all information of what went wrong. But to be honest I'm not sure which would be best either way, I haven't got a strong preference.

Reply:
me neither, just wondering

Reply:
I'd say the semantics of this cause is slightly different from the semantics of `__cause__`, because it's not just the exception that caused this exception, it's more part of an error we're collection information about. But the difference is so slight (and arguably subjective) that I think either option would be fine.